api-docs/apiLive.htm

```diff
@@ -12906,6 +12906,16 @@ <h2 class="flybar-button">Template</h2>
 		"maxValue":"20"
 	}
 ],
+"loanProductConfigurableAttributes":[
+	{configureAmortization : true,	
+	 configureInterestMethod : true,
+	 configureRepaymentStrategy : false,
+	 configureInterestCalcPeriod : true,
+	 configureArrearsTolerance : false,
+	 configureRepaymentFrequency : true,
+	 configureMoratorium : true,
+	 configureGraceOnArrearsAging : true}
```

Comment:
all attribute names should be the same as the names in the loan product documentation